plutus-core/plutus-core/src/PlutusCore/Parser/Type.hs

```diff
@@ -1,16 +1,25 @@
+{-# LANGUAGE GADTs             #-}
+{-# LANGUAGE LambdaCase        #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PolyKinds         #-}
+{-# LANGUAGE TypeApplications  #-}
 
 module PlutusCore.Parser.Type where
 
 import PlutusPrelude
-import Text.Megaparsec hiding (ParseError, State, parse, some)
 
 import PlutusCore.Core.Type
+import PlutusCore.Data
 import PlutusCore.Default
 import PlutusCore.MkPlc (mkIterTyApp)
 import PlutusCore.Name
 import PlutusCore.Parser.ParserCommon
 
+import Control.Monad
+import Data.ByteString (ByteString)
+import Data.Text (Text)
+import Text.Megaparsec hiding (ParseError, State, many, parse, some)
+
 -- | A PLC @Type@ to be parsed. ATM the parser only works
 -- for types in the @DefaultUni@ with @DefaultFun@.
 type PType = Type TyName DefaultUni SourcePos
```

Comment:
No idea.

Reply:
It stands for parsed type/term, as explained in the Haddock, to emphasize that they are not any type/term, because only a subset of them are parsable. I really should stop abbreviating the names. Sorry about the confusion.